### DISCUSSION TITLE
Toutes rondes ou doubles rondes

### DISCUSSION THREAD

**Discussion by wilchel** [pascalaubry/papi-web#39] [Ideas]
Bonjour,

J'ai arbitré le championnat jeunes ce week-end, et dans mon tournoi, nous utilisions un système d'appariement toutes rondes et double rondes. L'un des avantages de Papi Web est qu'il permet aux joueurs de vérifier si les résultats saisis sont corrects.

Cependant, j'ai remarqué qu'il n'était pas possible de vérifier les résultats une fois le dernier match saisi. L'affichage bascule automatiquement sur la prochaine ronde, sans possibilité de revenir sur la ronde précédente. Je comprends que cela est dû au fait que Papi précalcule toutes les rondes.

Je propose donc d'ajouter un sélecteur de rondes sur Papi Web, qui permettrait de visualiser les rondes précédentes. Par défaut, ce sélecteur pourrait être positionné sur "Dernière ronde en cours", offrant ainsi plus de flexibilité sans interrompre le déroulement du tournoi.

**Comment by pascalaubry** [comment]
Bonjour Bachir,
Cela sera résolu dans une prochaine version qui permettra d'apparier en mode brouillon avant de publier officiellement les appariements (fonctionnalité non présente dans Papi).
Stay tuned...
